### DISCUSSION TITLE
Config option for MYSQL_ATTR_LOCAL_INFILE in database settings

### DISCUSSION THREAD

**Discussion by davidhyland** [ExpressionEngine/ExpressionEngine#2989] [Feature Requests]
I have a use case for importing a large CSV file into a custom table using LOAD DATA LOCAL INFILE but where this worked in EE2 it no longer works in EE5 throwing "PDO::query(): LOAD DATA LOCAL INFILE forbidden" errors.

By manually adding PDO::MYSQL_ATTR_LOCAL_INFILE => true into the PDO connection options in mysqli_connection.php I was able to get around the issue but core hacks are never a good idea. So I propose a new custom config option whereby this can be set to TRUE.

I thank you!

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/3716/files will get that covered
